Header header
PoleClass[] pole_classes
